cdap-ui/app/cdap/components/Administration/AdminConfigTabContent/index.js

```diff
@@ -0,0 +1,113 @@
+/*
+* Copyright © 2018 Cask Data, Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may not
+* use this file except in compliance with the License. You may obtain a copy of
+* the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+* License for the specific language governing permissions and limitations under
+* the License.
+*/
+
+import React, {Component} from 'react';
+import ReloadSystemArtifacts from 'components/Administration/AdminConfigTabContent/ReloadSystemArtifacts';
+import NamespacesAccordion from 'components/Administration/AdminConfigTabContent/NamespacesAccordion';
+import SystemProfilesAccordion from 'components/Administration/AdminConfigTabContent/SystemProfilesAccordion';
+import SystemPrefsAccordion from 'components/Administration/AdminConfigTabContent/SystemPrefsAccordion';
+import LoadingSVGCentered from 'components/LoadingSVGCentered';
+import {MyNamespaceApi} from 'api/namespace';
+import {MyPreferenceApi} from 'api/preference';
+import {MyProfileApi} from 'api/cloud';
+import {Observable} from 'rxjs/Observable';
+import {Link} from 'react-router-dom';
+import T from 'i18n-react';
+
+require('./AdminConfigTabContent.scss');
+
+export const ADMIN_CONFIG_ACCORDIONS = {
+  namespaces: 'NAMESPACES',
+  systemProfiles: 'SYSTEM_PROFILES',
+  systemPrefs: 'SYSTEM_PREFS'
+};
+
+export default class AdminConfigTabContent extends Component {
+  state = {
+    namespaces: 0,
+    systemProfiles: 0,
+    systemPrefs: 0,
+    loading: true,
+    expandedAccordion: null
+  };
+
+  expandAccordion = (accordion) => {
+    if (this.state.expandedAccordion === accordion) {
+      this.setState({
+        expandedAccordion: null
+      });
+    } else {
+      this.setState({
+        expandedAccordion: accordion
+      });
+    }
+  }
+
+  componentWillMount() {
+    Observable
```

Comment:
Would be good if we can parallelize these calls so that we can right away render the accordions and show contents as and when it arrives.